### PR TITLE
adding bandinstown api js

### PR DESCRIPTION
**Pull request by tonyphak**
check out my bands in town api

### CODE REVIEW

File: assets/javascript/app.js [hunk 1]
```diff
@@ -8,4 +8,24 @@ var config = {
     storageBucket: "gig-it-fafd1.appspot.com",
     messagingSenderId: "660146414559"
   };
-  firebase.initializeApp(config);
\ No newline at end of file
+<<<<<<< Updated upstream
```
Comment (ptamayo97) [inline assets/javascript/app.js]: delete this line

File: assets/javascript/app.js [hunk 2]
```diff
@@ -8,4 +8,24 @@ var config = {
...
+  firebase.initializeApp(config);
+=======
```
Comment (ptamayo97) [inline assets/javascript/app.js]: delete this line

File: assets/javascript/app.js [hunk 3]
```diff
@@ -8,4 +8,24 @@ var config = {
...
+  firebase.initializeApp(config);
```
Comment (ptamayo97) [inline assets/javascript/app.js]: repeated line

File: assets/javascript/app.js [hunk 4]
```diff
@@ -8,4 +8,24 @@ var config = {
...
+    url: queryURL,
+    method: "GET"
+  }).then(function(response) {
+  console.log(response);
+  
+  })
+ }
+ //call back function to display JSON for artist and events
+ displayArtistInfo()
+>>>>>>> Stashed changes
```
Comment (ptamayo97) [inline assets/javascript/app.js]: delete this line

File: assets/javascript/app.js (hunk 3 above)
Comment (tonyphak) [reply assets/javascript/app.js]: i have deleted this duplicate line

**Comment by tonyphak** [comment]
Hey Paul recheck my app.js to see if code is okay

**Review by ptamayo97 — APPROVED** [review]

**Pull request merged**